### ISSUE TITLE
nested comments confuse Proof General (but not the client)

### ISSUE THREAD

**Issue by alleystoughton** [easyuc/EasyUC#43]
nested comments confuse Proof General (but not the client)

**Comment by alleystoughton** [comment]
EasyCrypt's Proof General script handles this by setting a single local variable. But it also doesn't use the proof-easy-config easy configuration mechanism. Not clear if setting this variable in some hook is enough to get the right behavior.

This issue is of moderate priority, as it's common to want to comment-out a part of a script, temporarily, and that part may have comments.

**Comment by 01tomislav** [comment]
apparently GitHub also has problems with nested comments (*"(**)"*)

**Comment by 01tomislav** [comment]
should look like this (\*\"(**)\"\*)

**Comment by alleystoughton** [comment]
Yes, this may not be fixable while using proof-easy-config. I tried setting the variable by hand in the script buffer, but it has no effect.
